action.yml

```diff
@@ -28,6 +28,9 @@ inputs:
   tags:
     description: 'Stringified form of a JSON object with the following shape: { [propertyName: string]: string; }'
     required: false
+  contentType:

```

Comment:
Is there a version field? If we could bump the minor version that would be great. That way we can signify a new parameter is available.

Reply:
Versioning for GitHub Actions uses the tag associated with the GitHub release, and yes, I'll bumping the minor version when I create the new release/tag. If interested, the full versioning details are described [here](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md).

Reply:
Sounds great, thanks.